src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
```

Comment:
setting 120px as a padding is not a good idea, since it will have unwanted effects in mobile devices that would see our website.

a suggestion is using scss's functions to calculate the width of screen, and then setting the padding accordingly, even though the proposed design does not include a mobile version for reference, it is a good practice to consider how the website will be viewed on different viewports.